pkgs/development/python-modules/psycopg/default.nix

```diff
@@ -87,6 +87,10 @@ let
       tomli
     ];
 
+    buildInputs = [
+      postgresql
+    ];
```

Comment:
Can you elaborate why this is now needed?

Reply:
This is just a proper place for this as a library dependency. Ideally, we would remove it from `nativeBuildInputs` but that will probably not work until https://github.com/psycopg/psycopg2/issues/1001. cc @SuperSandro2000 who made this change.

Reply:
Yes, IIRC they use pg_config

Reply:
Added a comment.